tests/test_configuration.py

```diff
@@ -14,164 +14,150 @@
 """
 Unit tests for the :mod:`pennylane` configuration classe :class:`Configuration`.
 """
-# pylint: disable=protected-access
-import unittest
+import pytest
 import os
 import logging as log
 
 import toml
 
-from defaults import pennylane, BaseTest
+from defaults import pennylane
```

Comment:
As we complete the move to pytest, we will eventually delete the `defaults.py` file.

Since developers need to install PennyLane via `pip install -e .` due to the use of entry points, it should be fine to simply have `import pennylane` here.